cluster/manifests/prometheus/statefulset.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  annotations:
+  labels:
+    application: prometheus
```

Comment:
We should have the `version` label here to be consistent with the other manifests. :)